modules/distr-tracing-config-security-ossm-cli.adoc

```diff
@@ -35,7 +35,7 @@ $ oc project istio-system
 +
 [source,terminal]
 ----
-$ oc edit -n tracing-system -f jaeger.yaml
+$ oc edit Jaeger jaeger
 ----
```

Comment:
**QE:** Is my change the proper way to edit the Jaeger custom resource file? It is what worked for me. For ex, the tracing-system namespace doesn't exist.